test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -445,6 +455,7 @@ asyncTest('hammerhead functions should not be in strict mode (GH-344)', function
 
     eventSimulator.click(button[0]);
 });
+
 asyncTest('should not define window.event property if event is raised in iframe for element of top window', function () {
```

Comment:
asyncTest('should not define the window.event property if the event is raised in iframe for the element of top window'